packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,139 @@
+import { ECIES, PhoneNumberUtils, SignatureUtils } from '@celo/utils'
 import BigNumber from 'bignumber.js'
+import { zip } from 'lodash'
+import * as Web3Utils from 'web3-utils'
 import { Attestations } from '../generated/types/Attestations'
-import { BaseWrapper, proxySend } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend, toNumber, tupleParser, wrapSend } from './BaseWrapper'
+import { CeloToken } from './TokenWrapper'
```

Comment:
I don't think we need this: we already have:
```ts
export type CeloToken = CeloContract.GoldToken | CeloContract.StableToken
```
which are enums (strings). and you can use those to get an address using `kit.registry`

Reply:
Do you think that consumers should pass in a string then instead of the contract itself?